docs/reference/api/platform/Time.md

```diff
@@ -1,8 +1,16 @@
 ## Time
 
-The C date and time functions are a group of functions in the standard library of the C programming language implementing date and time manipulation operations. They provide support for time acquisition, conversion between date formats and formatted output to strings.
+The C `date` and `time` functions are a group of functions in the standard library of the C programming language implementing date and time manipulation operations. They provide support for time acquisition, conversion between date formats and formatted output to strings.
 
-### Time hello, world
+You can convert time to a human-readable format using `ctime`, `localtime`, `strftime` and other C standard functions.
+
+You cannot use `mktime` and `localtime` C standard library functions in an interrupt handler with the GCC toolchain. We have added dedicated routines for `mktime` and `localtime`, which are optimized and faster then C standard library functions, to overcome this issue.
```

Comment:
I dont understand this sentence. Did we add new functions with the same name?

Reply:
No we added same functions as `_rtc_mktime`, `_rtc_localtime`. Will update the doc with new name